dbms/src/Flash/Disaggregated/WNFetchPagesStreamWriter.cpp

```diff
@@ -39,12 +39,8 @@ WNFetchPagesStreamWriterPtr WNFetchPagesStreamWriter::build(
     const PageIdU64s & read_page_ids,
     UInt64 packet_limit_size)
 {
-    return std::unique_ptr<WNFetchPagesStreamWriter>(new WNFetchPagesStreamWriter(
-        task.seg_task,
-        task.column_defines,
-        task.output_field_types,
-        read_page_ids,
-        packet_limit_size));
+    return std::unique_ptr<WNFetchPagesStreamWriter>(
+        new WNFetchPagesStreamWriter(task.seg_task, task.column_defines, read_page_ids, packet_limit_size));
```

Comment:
Can we use `make_unique` here?

Reply:
No because we define the ctor function with "private"